app/models/pattern_detection_log.py

```
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, BigInteger, Float, Boolean, DateTime, ForeignKey, Enum as SqlEnum, func
```

Ruff (0.12.2)

`sqlalchemy.func` imported but unused

Remove unused import: `sqlalchemy.func`

(F401)